java/java.source.base/test/unit/src/org/netbeans/api/java/source/gen/ClassMemberTest.java

```diff
@@ -1845,12 +1847,44 @@ public void run(WorkingCopy workingCopy) throws Exception {
         assertEquals(golden, res);
     }
 
+    public void testRemoveMethodFromUnnamed() throws Exception {
+        testFile = new File(getWorkDir(), "Test.java");
+        TestUtilities.copyStringToFile(testFile,
+            "void test() {\n" +
+            "}\n"
+            );
+        String golden =
+            "";
+
+        sourceLevel = "21";
```

Comment:
same issue here. Value will depend on test order.

Reply:
I don't think there should be any dependence on test order. Every test gets its own instanceof of the class (and its own working directory), I believe, and hence every test starts with `sourceLevel == null`, which is (was) the default so far. If the test sets a source level value, it should be used only for that test, and no other.